### DISCUSSION TITLE
Auth0 provider

### DISCUSSION THREAD

**Discussion by sophie-warner-annalise-ai** [opentofu/opentofu#1975] [General]
Hi, looks like the Auht0 provider hasn't had much love since it was forked https://github.com/opentofu/terraform-provider-auth0
Could someone turn issues so that it can be worked on?
- Readme has not been updated since it was forked.
- Doesn't run on apple silicon

**Comment by ghost** [comment]
Hey @sophie-warner-annalise-ai the providers are still under the MPL license on the HashiCorp side. The only reason for us to maintain a fork is to build the binaries needed for OpenTofu, which HashiCorp doesn't publish, but we don't add fixes or changes to these forks to avoid a split in functionality.

The auth0 provider specifically has been archived on the HashiCorp side in favor of [this provider](https://github.com/auth0/terraform-provider-auth0), so I don't think it makes sense for us to patch the fork. Rather, we should archive ours as well since it isn't getting any more patches and encourage people to use the official auth0 provider.

I opened https://github.com/opentofu/opentofu/issues/1976 to that effect.

**Reply by sophie-warner-annalise-ai** [reply]
Thanks for your prompt response. 

Is there a plan to fork the new provider into opentofu?

**Reply by ghost** [reply]
@sophie-warner-annalise-ai there is no need to, Terraform providers are compatible with OpenTofu and the new provider is available in the OpenTofu registry. You can use the new provider as-is by referencing it in your config as described [in their documentation](https://github.com/auth0/terraform-provider-auth0#documentation):

```hcl
terraform {
  required_providers {
    auth0 = {
      source  = "auth0/auth0"
      version = ">= 1.0.0" # Refer to docs for latest version
    }
  }
}

provider "auth0" {}
```
